src/components/Bookings.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from "react";
+import Search from "./Search.js";
+import SearchResults from "./SearchResults.jsx";
+import AddCustomerForm from "./AddCustomerForm";
+import "./Bookings.css";
+
+const Bookings = () => {
+  const [Booking, setBookings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+
+  console.log(loading);
+  console.log(Booking);
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me/delayed`)
+      .then(res => res.json())
+      .then(data => {
+        if (data.error) {
+          throw data;
```

Comment:
Here you're throwing an error if the api request is unsuccessful and this is problematic for 2 reasons:
- The `throw` means the app will crash and stop working
- You won't hit the other bit of error handling you did in your `catch` block